lib/sentry/sources.ex

```diff
@@ -66,12 +66,10 @@ defmodule Sentry.Sources do
   @type file_map :: %{pos_integer() => String.t()}
   @type source_map :: %{String.t() => file_map}
 
-  def load_files do
-    Enum.reduce(
-      Config.root_source_code_paths(),
-      %{},
-      &load_files_for_root_path/2
-    )
+  @doc false
+  @spec load_files([Path.t()]) :: source_map()
+  def load_files(paths \\ Config.root_source_code_paths()) do
```

Comment:
If we default to `Config.root_source_code_paths/0`, then we can have tests that don't have to modify the env, and it'll work better when revamping configs, because this is really a compile-time config.